### PR TITLE
Adding JSON Schemas for XDM Business Person

### PR DESCRIPTION
**Pull request by mjasrotia**
Please link to the issue #…

### CODE REVIEW

File: components/mixins/profile/b2b-person-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+          "title": "Contact Account id",
+          "description": "ID of the account that’s the parent of this person.",
+          "type": "string"
+        },
+        "xdm:personOwnerID": {
+          "title": "Person Owner ID",
+          "description": "Person owner id.",
+          "type": "string"
+        },
+        "xdm:personEmail": {
```
Comment (cmathis) [inline components/mixins/profile/b2b-person-details.schema.json:49]: We already have a standard field for Profile named xdm:personalEmail that comes from the profile-personal-details mixin. This field should be renamed to "xdm:personalEmail" or you could already reference and use that mixin.

**Comment by cmathis** [comment]
I hope we are not overloading the "Individual Profile" class to mean too many things. We've traditionally used this class to represent B2C and more traditional CRM type use-cases. If the B2B use-case is completely different with little overlap in the existing Profile fields, then we may want to consider  a new "B2B Profile" class.

File: components/mixins/profile/b2b-person-details.schema.json (hunk 1 above)
Comment (mjasrotia) [reply components/mixins/profile/b2b-person-details.schema.json]: I did notice this so, I removed this already.

File: components/mixins/profile/b2b-person-details.schema.json [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          "description": "Data.com key",
+          "type": "string"
+        },
+        "xdm:lastTransferDate": {
+          "title": "Last Transfer Date",
+          "description": "Last transfer date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:unReadByOwner": {
```
[the remark this answers is not in the document]
Comment (mjasrotia) [reply components/mixins/profile/b2b-person-details.schema.json:154]: Will remove

File: components/mixins/profile/b2b-person-details.schema.json [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+          "title": "Jigsaw Person Id",
+          "description": "Person’s Data.com ID if available.",
+          "type": "string"
+        },
+        "xdm:jigsaw": {
+          "title": "Jigsaw",
+          "description": "Data.com key",
+          "type": "string"
+        },
+        "xdm:lastTransferDate": {
```
[the remark this answers is not in the document]
Comment (mjasrotia) [reply components/mixins/profile/b2b-person-details.schema.json:148]: This attribute is in Salesforce.

File: components/mixins/profile/b2b-person-details.schema.json [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+          "format": "date-time"
+        },
+        "xdm:unReadByOwner": {
+          "title": "Read By Owner Indicator",
+          "description": "Read by owner indicator.",
+          "type": "boolean"
+        },
+        "xdm:personSource": {
+          "title": "Person Source",
+          "description": "Source from where the information regarding the person was recieved.",
```
[the remark this answers is not in the document]
Comment (mjasrotia) [reply components/mixins/profile/b2b-person-details.schema.json:161]: Will fix

File: components/mixins/profile/b2b-person-components.schema.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+              },
+              "xdm:sourceConvertedContactID": {
+                "title": "Source Converted Contact ID",
+                "description": "Source converted contact ID.",
+                "$ref": "https://ns.adobe.com/xdm/datatypes/b2b-source"
+              },
+              "xdm:personSegmentTraits": {
+                "title": "Person Segmentation Attributes",
+                "description": "These attributes are required to build predicates for person segmentation.",
+                "$ref": "https://ns.adobe.com/xdm/mixins/b2b-person-details#/definitions/b2b-person-segment-traits"
```
[the remark this answers is not in the document]
Comment (mjasrotia) [reply components/mixins/profile/b2b-person-components.schema.json:57]: personalEmail is present in the "profile-personal-details" mixin. the root level property will be available from this mixin. I will add this as additional attribute in the array.

File: components/mixins/profile/b2b-person-details.schema.json [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+        "xdm:convertedOpportunityID": {
+          "title": "Converted Opportunity ID",
+          "description": "Opportunity Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:jigsawPersonID": {
+          "title": "Jigsaw Person Id",
+          "description": "Person’s Data.com ID if available.",
+          "type": "string"
+        },
```
[the remark this answers is not in the document]
Comment (mjasrotia) [reply components/mixins/profile/b2b-person-details.schema.json:142]: Will remove

File: components/mixins/profile/b2b-person-components.schema.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (peichiang) [reply components/mixins/profile/b2b-person-components.schema.json:61]: this "email" is probably the "work" email than a "personal" email.

[the remark this answers is not in the document]
Comment (peichiang) [reply components/mixins/profile/b2b-person-components.schema.json:61]: saw you added "xdm:personEmail". thank you!

**Comment by peichiang** [comment]
Manik, Thank you very much for your quick response. This looks good to me.

**Review by cmathis — APPROVED** [review]

File: components/mixins/profile/b2b-person-components.schema.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (cmathis) [reply components/mixins/profile/b2b-person-components.schema.json:61]: Guys, we can't have xdm:personEmail and xdm:personalEmail. What is the difference?

**Pull request merged**